static/tests/frontend/specs/api-dropdownCaretElementChanged.js

```diff
@@ -22,31 +22,32 @@ describe("ep_script_elements - dropdown", function(){
     this.timeout(60000);
   });
 
-  it("changes option select when script element is changed", function(done) {
+  it("changes the caret element when api triggers that dropdown changed", function(done) {
```

Comment:
Que tal "changes the ~caret element~ **line type** when api ~triggers that dropdown changed~ **receives a message that ...**". Não temos mais dropdown nenhum aqui, não dá para mencionar um.